### DISCUSSION TITLE
Error "insufficient Funds Available To Construct Transaction"

### DISCUSSION THREAD

**Discussion by jordillonchesteve** [Ride-The-Lightning/RTL#1338] [General]
Hi, I am trying to send the all the funds on-chain to another on-chain bitcoin address but I get this error:

![image](https://github.com/Ride-The-Lightning/RTL/assets/7152949/130047ac-ef06-4b46-b5b2-fa432c07e45d)

If I do a sweep all, will I send all this funds or will it also sweep my lightning funds? Is there any other way to send all the on-chain funds?

Thank you.

**Comment by saubyk** [comment]
Hi @jordillonchesteve are you on a LND node? If yes and if you're looking to sweep out all of your onchain funds, you should use `Sweep All`.

> If I do a sweep all, will I send all this funds or will it also sweep my lightning funds?

No. Lightning funds (as in the funds in lightning channels) are not swept with sweep all. if you intend to sweep funds in the lightning channels you'd need to close the lightning channels first

Also note that after the sweep, some onchain funds would still be left on the wallet (if you have open lightning channels). This is required to ensure that some funds are available to sweep anchor outputs after channel closure

**Reply by jordillonchesteve** [reply]
Thank you very much for your explanation, yes I am on a LND node. I am using the myNode all-in-one solution.
